### ISSUE TITLE
Support custom character conditions?

### ISSUE THREAD

**Issue by cmfcmf** [cmfcmf/roc-and-roll#140]
They could simply use a question mark icon and have a configurable tooltip.

**Comment by cmfcmf** [comment]
> I think it could be fun to use Emoji here as well, to provide people with more choices for their own conditions. (I think we already have a lot where it could make sense slowed, polymorphed, cursed, hunters mark, sickened etc.)

True!

> For uniformity it would however be cool to use one specific font so it looks the same across systems.

Indeed! This also applies to the reactions feature. It would be cool if reactions looked the same on every system.

> I found this open source project which I found quite pretty: [openmoji.org](https://openmoji.org/) There is also an (experimental) black and white version that would not introduce as much noise.

I love it 👍 I don't even know whether custom conditions should be black and white, I think color would be fine for them.


> Obviously they are not as well matched to the conditions (disarmed or unconscious especially are difficult), but I think that's also true for the ones we are using at the moment. Whenever we want something that is not exactly what is designed for it becomes difficult.

True - I would vote for keeping the existing condition icons we have, and only replace the subset of them for which we find a better-suited emoji.

> I think (if we chose to implement this) it would make sense to still have several predefined conditions, and then add the options to add custom ones to the suggestion list to be able to reuse them, maybe even across players in one game.

Definitely 👍

**Comment by corinnaj** [comment]
> > Obviously they are not as well matched to the conditions (disarmed or unconscious especially are difficult), but I think that's also true for the ones we are using at the moment. Whenever we want something that is not exactly what is designed for it becomes difficult.
> 
> True - I would vote for keeping the existing condition icons we have, and only replace the subset of them for which we find a better-suited emoji.

I'm a bit hesitant about having both types of icons. I think we should reduce the number of visual styles that are in the application by default as much as we can, since users will also add their own images etc. Right now we already have Emoji (although we might only have one type here after this), FontAwesome and often the auto generated token images...

**Comment by cmfcmf** [comment]
> > > Obviously they are not as well matched to the conditions (disarmed or unconscious especially are difficult), but I think that's also true for the ones we are using at the moment. Whenever we want something that is not exactly what is designed for it becomes difficult.
> > 
> > 
> > True - I would vote for keeping the existing condition icons we have, and only replace the subset of them for which we find a better-suited emoji.
> 
> I'm a bit hesitant about having both types of icons. I think we should reduce the number of visual styles that are in the application by default as much as we can, since users will also add their own images etc. Right now we already have Emoji (although we might only have one type here after this), FontAwesome and often the auto generated token images...

You're right in that I would also love to have a more consistent design! I guess what I was trying to say is that I would be fine to have inconsistent styling if you consider the trade-off of the inconsistent styling to be lower than the trade-off of potentially worse icons for certain conditions. But I'm fine either way 👍
